### PR TITLE
Replace ghetzel

### PR DESCRIPTION
**Pull request by Jeff-Moorhead-Pline**

### CODE REVIEW

File: diecast [hunk 1]
```diff
@@ -1 +0,0 @@
-cmd/diecast
```
Comment (asalatto) [inline diecast:1]: What was this? Sure it's not actually doing anything? #mysteries #garyverse

**Review by dwhite-performline — APPROVED** [review]

**Review by john-pl — APPROVED** [review]
I like the branch name

**Review by asalatto — APPROVED** [review]
godspeed, glory

File: diecast (hunk 1 above)
Comment (Jeff-Moorhead-Pline) [reply diecast:1]: that was a symlink to a compiled version of this project. Totally unnecessary and should never have been checked in to begin with.

**Pull request merged**